Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -430,8 +430,8 @@ def audit_deps
             problem "Dependency '#{dep.name}' is an alias; use the canonical name '#{dep.to_formula.full_name}'."
           end
 
-          if @new_formula && dep_f.keg_only_reason &&
-             !["openssl", "apr", "apr-util"].include?(dep.name) &&
+          if @official_tap && @new_formula && dep_f.keg_only_reason &&
+             !["openssl", "apr", "apr-util", "openblas"].include?(dep.name) &&
```

Comment:
This change permits all keg-only dependencies for non-core taps. We want to permit `openblas` specifically on non-core taps. I'd suggest…
```ruby
&& !(["openblas"].include?(dep.name) && !@official_tap)
```
(unttested, I hope that's right)

Reply:
Equivalently (whichever is more readable)
```ruby
&& (!["openblas"].include?(dep.name) || @official_tap)
```

Reply:
Duh, sorry. Long day.